evap/evaluation/models.py

```diff
@@ -1062,6 +1062,7 @@ def send_to_users_in_courses(cls, template, courses, recipient_groups, use_cc):
             # Collect recipients for all courses.
             # Don't include delegates and CC users of the responsible person of a course to the recipient list of this
             # course, because they will get the notification in CC anyway.
+            # TODO why only for the responsible, not for all contributors
```

Comment:
and you will address this in the near future? i would prefer having a ticket over TODOs in the code ;)